sdk/communication/communication-phone-numbers/src/utils/mappers.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { HttpOperationResponse, HttpResponse } from "@azure/core-http";
+import { WithResponse } from "..";
+
+/**
+ * Attach http response to a model
+ */
+export const attachHttpResponse = <T>(
```

Comment:
We will have to remove this in a separate PR to no longer expose the raw response.